infrastructure/src/main/java/org/corfudb/infrastructure/DataStore.java

```diff
@@ -36,6 +36,7 @@
 
     private final Map<String, Object> opts;
     private final boolean isPersistent;
+    @Getter
```

Comment:
It's used once, by ServerContext.  However, you ask about another @Getter annotation added in  SequencerServer that I'll remove (assuming the compiler agrees with my eyes).